models/labels/addresses/__single_category_labels__/quests/persona/coinbase_wallet_quests/labels_optimism_coinbase_wallet_quest_participants.sql

```diff
@@ -17,13 +18,13 @@ with
 select
   blockchain,
   quester_address AS address,
-  "Coinbase Wallet Quest Participant" AS name,
-  "quests" AS category,
-  "msilb7" AS contributor,
-  "query" AS source,
-  timestamp('2023-03-11') as created_at,
+  'Coinbase Wallet Quest Participant' AS name,
+  'quests' AS category,
+  'msilb7' AS contributor,
+  'query' AS source,
+  from_iso8601_timestamp('2023-03-11') as created_at,
```

Comment:
@jkylling can you help us determine if this is the best approach for hardcoded timestamps?

@MSilb7 feel free to follow up on this comment with more context, but from what i gather, on the dune UI it works to do `timestamp '2023-03-11'` but on spellbook, it errors out

Reply:
as noted in legacy file, the spark approach was:
`timestamp('2023-03-11') as created_at,`

Reply:
`cast('2023-03-11' as timestamp)` also works in Dune UI, but errors out in Spellbook

Reply:
My read is that Dune UI accepts timestamps without timezones, but spellbook needs a timezone.